utils.py

```diff
@@ -1075,8 +1085,14 @@ def label_multiline(layout, text='', icon='NONE', width=-1, max_lines=10, split_
     li = 0
     for l in lines:
```

Comment:
I would suggest:
```
for index, line in enumerate(lines):
```

- using `enumerate()` we can remove `li=0` and `li += 1`
- renaming `li` to `index` makes it more readable
- renaming `l` to `line` is just better

Reply:
we can't replace li - line index, it can increase also lower in the while loop. I'll rename the vars.